### PR TITLE
Allow paragonie/random_compat 9.99.99

### PR DESCRIPTION
**Pull request by snapshotpl**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ before_install:
   - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
 
 install:
-  - travis_retry composer install $COMPOSER_ARGS --ignore-platform-reqs
+  - travis_retry composer install $COMPOSER_ARGS
```
Comment (michalbundyra) [inline .travis.yml]: Revert this and add `paragonie/random_compat` into `LEGACY_DEPS` on PHP 5.6 locked

Comment (snapshotpl) [reply .travis.yml]: @webimpress :+1:

**Comment by snapshotpl** [comment]
@webimpress looks like travis issue. Can you restart this build?

**Comment by michalbundyra** [comment]
@snapshotpl 

> looks like travis issue. Can you restart this build?

I can't, sorry. You have right - travis issue.

It is fine for me now 👍

**Review by michalbundyra — APPROVED** [review]
LGTM 👍

**Comment by michalbundyra** [comment]
@weierophinney @Ocramius can we have it merged and released, please?

**Comment by weierophinney** [comment]
Can somebody explain to me what the 9.99.99 version is, exactly, and why this is required? I'd like to know for the changelog...

**Comment by michalbundyra** [comment]
@weierophinney 

> Can somebody explain to me what the 9.99.99 version is, exactly, and why this is required? I'd like to know for the changelog...

I think here we have answer:
https://github.com/paragonie/random_compat/commit/7dd806d6be2dd16a43a9ee05b87255c8c52bc7a4

And maybe also worth to check this:
https://github.com/paragonie/random_compat/compare/v2.0.17...v9.99.99

**Comment by weierophinney** [comment]
Cherry-picked to develop for release with 3.2.0.

**Pull request closed without merging**